### PR TITLE
#36324 - update header HTML fits to current version of alpaca module

### PR DESCRIPTION
**Pull request by szafran89**

### CODE REVIEW

File: Magento_Theme/templates/header/marketing-bar.phtml [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
-            type="button"
-            aria-label="<?= /* @escapeNotVerified */ __('Close marketing bar') ?>"
-    >
-        <svg title="close" class="marketing-bar__close-icon">
-            <use xlink:href="<?= $block->getViewFileUrl('/images/icons-sprite.svg#close'); ?>"></use>
-        </svg>
-    </button>
+    <div class="container marketing-bar__container">
+        <div class="marketing-bar__text">
+            <!-- TODO: Move to CMS block -->
```
Comment (kruchy8) [inline Magento_Theme/templates/header/marketing-bar.phtml]: comment

File: Magento_Theme/templates/header/marketing-bar.phtml [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
+            Subscribe and get 10$ for Shopping!
```
Comment (kruchy8) [inline Magento_Theme/templates/header/marketing-bar.phtml]: Shouldn't this go from translate function ?

File: Magento_Theme/templates/header/header.phtml [hunk 3]
```diff
@@ -1,43 +1,49 @@
...
+                         class="
+                             icon
+                             header-button__icon
+                             header-button__icon--small
+                             header-button__icon--open
+                         "
+                    >
+                        <use xlink:href="<?= $block->getViewFileUrl('/images/icons-sprite.svg#mobile-menu') ?>"></use>
+                    </svg>
+                    <svg title="<?= /* @escapeNotVerified */ __('Side menu close') ?>"
```
Comment (kruchy8) [inline Magento_Theme/templates/header/header.phtml]: Svg have no title attribute here (https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title)

File: Magento_Theme/templates/header/header.phtml [hunk 4]
```diff
@@ -1,43 +1,49 @@
...
-                            <?= __('menu') ?>
-                        </span>
-                    </div>
+                <button class="
+                            header-button
+                            header-button--side-menu
+                            header-button--reset-margin
+                        "
+                        type="button"
+                >
```
Comment (kruchy8) [inline Magento_Theme/templates/header/header.phtml]: aria attribute missing

**Comment by kruchy8** [comment]
@szafran89 please fix title attribute issue of svg tags used in code within this issue

File: Magento_Theme/templates/header/marketing-bar.phtml (hunk 2 above)
Comment (Igloczek) [reply Magento_Theme/templates/header/marketing-bar.phtml]: That's why it's marked as "move to CMS block", b/c it's just placehoder

**Comment by Igloczek** [comment]
Merged to develop

**Pull request closed without merging**